toolchest_client/api/query.py

```diff
@@ -50,6 +51,10 @@ def __init__(self):
         self.thread_name = ''
         self.thread_statuses = None
 
+        self.presigned_s3_url = None
+        self.unpacked_output = None
+        self.output = output_object if output_object else Output()
```

Comment:
Is there a case in which the `output_object` is passed during initialization?

Reply:
(and where we want to set `self.output` with it?)

Reply:
Yes; when multithreading (which all of our jobs are currently doing), an `Output` is initialized before the `Thread()` call for each thread and stored in the `Tool.thread_outputs` dict. This is to provide persistence after `Query.run_query()` executes.

The behavior is currently to return the `Output` from the first thread, which might not make sense because its `output_path` leads to a temporary file outputted from a single thread. I'll mark this as an issue for the future, since we aren't currently using multithreading and may change threading behavior in the future fundamentally.